terra-scm-1.rockspec

```diff
@@ -0,0 +1,34 @@
+package = "terra"
+version = "scm-1"
+
+source = {
+   url = "git://github.com/zdevito/terra.git",
+   branch = "master",
+}
+
+description = {
+   summary = "A low-level counterpart to Lua.",
+   homepage = "",
+   license = "MIT",
+}
+
+build = {
+   type = "command",
+   build_command = [[
+   make clean
+
+    make LUAJIT_LIB=$(LUA_LIBDIR)/libluajit-5.1.a \
```

Comment:
I updated Terra so that changing the luajit used just requires setting the prefix, e.g.

    LUAJIT_PREFIX=/usr/local

This just needs a small modification to match, but I don't have a luarocks install to test it with.